### PR TITLE
Polishing all

### PR DESCRIPTION
**Pull request by guenk**

### CODE REVIEW

File: frontend/src/components/NavBarSuggest/style.scss [hunk 1]
```diff
@@ -95,12 +95,16 @@
...
+  color: #1f1f1f;
 }
 
 .active .text {
   animation: scale 0.5s linear forwards;
+  color: rgb(224, 224, 206);
+  font-size: 1em;
+  font-weight: bolder;
+
+  text-shadow: 2px 2px black;
```
Comment (Sax593) [inline frontend/src/components/NavBarSuggest/style.scss:107]: Unecessary space

File: frontend/src/components/Timeline/Style.scss [hunk 2]
```diff
@@ -24,21 +30,29 @@
...
   position: absolute;
   top: 30px;
-  left: 50%;
+  left: 52%;
   transform: translateX(-50%);
   background-color: #ddd;
   width: 100px;
   padding: 5px 10px;
+  margin-left: 5px;
+
```
Comment (Sax593) [inline frontend/src/components/Timeline/Style.scss]: Unecessary space

File: frontend/src/components/Timeline/Style.scss [hunk 3]
```diff
@@ -47,26 +61,32 @@
...
   width: 10px;
   height: 10px;
-  background-color: #ddd;
+  background-color: rgb(168, 163, 163);
   display: block;
   position: absolute;
   top: -8px;
-  margin-left: -2px;
   left: 50%;
+
```
Comment (Sax593) [inline frontend/src/components/Timeline/Style.scss:67]: Unecessary space

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Review by Sax593 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**